src/middlewared/middlewared/plugins/tunables.py

```diff
@@ -25,212 +26,110 @@ class TunableService(CRUDService):
     class Config:
         datastore = 'system.tunable'
         datastore_prefix = 'tun_'
-        datastore_extend = 'tunable.upper'
         cli_namespace = 'system.tunable'
 
-    def __init__(self, *args, **kwargs):
-        super(TunableService, self).__init__(*args, **kwargs)
-        self.__default_sysctl = {}
+    SYSTEM_TUNABLES = deque()
```

Comment:
Out of curiosity, any specific reason for deque ?

Reply:
Memory efficiency compared to a list as well as faster `.append` attribute